client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/dialog-styles.scss

```diff
@@ -11,9 +11,9 @@
 
 			.carrier-accounts__settings-cancel-dialog-title {
 				display: block;
-				margin: 0;
-				padding: 18px 28px;
+				margin: auto 28px;
```

Comment:
The title wasn't centered. Rather than hardcoding the top/bottom padding, I changed it to `margin-top`/`margin-bottom`: `auto`, so that it centers naturally.
If people still prefer `padding` to position the title, I could use flexbox instead.